fastvm/include/evm.h

```diff
@@ -449,6 +449,7 @@ enum evm_revision {
     EVM_BYZANTIUM = 4,
     EVM_AION = 5,
     EVM_CONSTANTINOPLE = 6,
+    EVM_AION_V1 = 7,
```

Comment:
I'd suggest we remove `EVM_CONSTANTINOPLE` becauses it's not implemented yet, and assign 6 to EVM_AION_V1.